use-cases/continual_learning/README.md

```diff
@@ -0,0 +1,125 @@
+# Continual Learning Demo
+
+## Description
+
+This software package delivers the building blocks required to develop an AI-powered weighing scale use case that continuously learns and refines its image classification model with incoming new data. This reference implementation provides an intuitive application that facilitates an image classification model to learn incrementally with new image samples (sourced either from local images folder or live streaming feed). The continual learning algorithm is based on Streaming Linear Discriminant Analysis (SLDA) [[1]](#references) enabling the model to learn from individual images without forgetting previously acquired knowledge, resulting in faster training performance with comparable accuracy (Top1/Top5). This solution is optimally suited for CPU utilization, ensuring its adaptability across diverse hardware setups. This software has been tested to work with Python 3.8.10.
```

Comment:
Is Python 3.8.10 the only working version? maybe re-work this a bit more like 3.8.10+ if that is working in general